@navikt/core/react/src/util/hooks/useMergeRefs.ts

```diff
@@ -0,0 +1,34 @@
+/* https://github.com/radix-ui/primitives/blob/main/packages/react/compose-refs/src/composeRefs.tsx */
+import React from "react";
+
+type PossibleRef<T> = React.Ref<T> | undefined;
+
+// https://github.com/gregberge/react-merge-refs
+/**
+ * Use `useMergeRefs`
+ * @internal
+ */
+export function mergeRefs<T>(
+  refs: Array<PossibleRef<T>>,
```

Comment:
Vi pleier vel å bruke `[]` i stedet for `Array<>`?
```suggestion
  refs: PossibleRef<T>[],
```